app/services/translation/sarvam_client.py

```
@@ -0,0 +1,68 @@
+        payload = {
+            "input": text,
+            "source_language_code": source_lang,
+            "target_language_code": target_lang,
+            "speaker_gender": "Male",
+            "mode": "formal",
+            "model": "sarvam-translate:v1"
+        }
+
+        for attempt in range(max_retries):
+            try:
+                response = requests.post(self.base_url, json=payload, headers=headers)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)